styles.css

```diff
@@ -101,3 +154,24 @@ p {
   color: blue;
   opacity: 0.7;
 }
+
+button {
+  cursor: pointer;
+}
+
+#contactsection {
+  display: none;
+  flex-direction: column;
+}
+
+.contact {
+  line-height: 22px;
+}
+
+#footer {
+  display: flex;
+  border: 2px solid blue;
+  width: 100%;
+  padding: 5px;
+  margin: 30px 0;
+}
```

Comment:
- [ ] [REQUIRED] The `footer` should be at the bottom of the web page like the [wireframe](https://github.com/microverseinc/curriculum-javascript/blob/main/books/images/awesome_books_full_website.png) `Microverse` is given!

![image](https://user-images.githubusercontent.com/63915024/191374814-25507818-adad-4357-979b-9ceac2cc69ef.png)